### ISSUE TITLE
Release a little beta for localizers to check all changes for 2.0

### ISSUE THREAD

**Issue by emmapeel2** [micahflee/onionshare#842]
Localizers were wondering if there is any possibility of making a beta release to check translations are OK. I think it would be nice to keep them on the loop always when there are major changes, and specially now (2.0) that many have translated the whole app for the first time to their languages.

**Comment by micahflee** [comment]
Yes, that's the [plan](https://github.com/micahflee/onionshare/issues/696)! I was hoping to have OnionShare in a more usable state and to be able to make a beta release way sooner, but OnionShare keeps hitting nasty bugs like #812 that I'm trying to get fixed before making that release.

Once the major issues left in the 2.0 milestone are finished, I'll make a development release, and give the translators a few weeks to work with it and update the translations (and also others a chance to work out any final bugs) before making the final release.

**Comment by emmapeel2** [comment]
Great! Just what localizers need! I will try to push for more translations before that.
